src/pudl/metadata/resources/eia.py

```diff
@@ -9,7 +9,23 @@
         "schema": {
             "fields": ["code", "label", "description"],
             "primary_key": ["code"],
-            "foreign_key_rules": {"fields": [["balancing_authority_code_eia"]]},
+            "foreign_key_rules": {
+                "fields": [["balancing_authority_code_eia"]],
+                "exclude": [
+                    "advanced_metering_infrastructure_eia861",
+                    "balancing_authority_eia861",
+                    "demand_response_eia861",
+                    "demand_response_water_heater_eia861",
+                    "dynamic_pricing_eia861",
+                    "energy_efficiency_eia861",
+                    "net_metering_customer_fuel_class_eia861",
+                    "net_metering_misc_eia861",
+                    "non_net_metering_customer_fuel_class_eia861",
+                    "non_net_metering_misc_eia861",
+                    "reliability_eia861",
+                    "sales_eia861",
```

Comment:
Still lots of work to do here, to get all the BA & Util IDs from EIA-861 into the entity tables.